packages/client/src/App.sass

```diff
@@ -24,11 +24,13 @@ $input-focus-border-color: transparent
 $input-border-color: black
 $input-shadow: none
 $button-background-color: transparent
-$button-border-color: black
+$button-border-color: #7464F0
```

Comment:
I believe this changes the button styles globally, is this to align with the new design?

Reply:
Yes!